### ISSUE TITLE
[Accessibility][Usable-Settings] : On Enabling 'Show Keys' button within Settings page, page gets automatically refreshed without providing any proper information. 

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2113]
**User Experience**
Users who rely on AT will face difficulty if  page gets automatically refreshed without providing any proper information. 

**Test Environment**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Select 'Settings' from left navigation.
5. Enable 'Show Keys' button and press tab key and observe the issue.

**Actual:**
On Enabling 'Show Keys' button within Settings page, page gets automatically refreshed without providing any proper information. Error pop-up will come and other page gets opened but screen reader does not provide any information.

**Expected:**
On Enabling 'Show Keys' button within Settings page, page should not get automatically refreshed and  it should provide the proper information related to error.

**Attachment:**

[Usable_Settings_Start keys.zip](https://github.com/microsoft/BotFramework-Composer/files/4261717/Usable_Settings_Start.keys.zip)

**Comment by liweitian** [comment]
@ShellySoni could you try this again with the updated master branch code? I cannot reproduce that error from my side.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
